packages/integration-tests/projects/connect-popup/env.js

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
Space at the beginning of the file is probably not required ?

Reply:
sure, removed 3f52008d0285d3d7d2561f1b41140adb82084304